msktutil.h

```diff
@@ -118,6 +118,21 @@
 #define MS_KERB_ENCTYPE_AES256_CTS_HMAC_SHA1_96 0
 #endif
 
+#define MS_KERB_DES_ENCTYPES \
```

Comment:
Sorry was not clear enough: 
The macro _definition_ should to have in braces , so you do not have to use braces when we use it.
```
#define MS_KERB_DES_ENCTYPES \
             (  MS_KERB_ENCTYPE_DES_CBC_CRC | \
                 MS_KERB_ENCTYPE_DES_CBC_MD5) 
```

Reply:
Ah sorry, I will do. Misunderstanding.

Reply:
Done!